UI/import_csv/import_csv.html

```diff
@@ -27,6 +27,7 @@
                  label = "_none_"
                  value = request.reference
                  size  = 15
+                 required = "true"
```

Comment:
These fields were not marked "Required" because they don't matter for GIFI/parts/overhead/services uploads. Maybe you can add an IF block around these inputs and a template variable to indicate whether or not they apply to the upload (like `request.multi`?)

Reply:
Please check the new commit